lib/api/cln/cln_client.dart

```diff
@@ -161,4 +163,26 @@ class CLNApi extends AppApi {
             jsonResponse as Map<String, dynamic>,
             snackCase: mode == ClientMode.unixSocket));
   }
+
+  @override
+  Future<AppListPays> listPays() {
+    dynamic params;
+    switch (mode) {
+      case ClientMode.grpc:
+        params = CLNListPaysRequest(grpcRequest: ListpaysRequest());
+        break;
+      case ClientMode.unixSocket:
+        params = CLNListPaysRequest(unixRequest: <String, dynamic>{});
```

Comment:
```suggestion
        params = CLNListPaysRequest(unixRequest: <String, dynamic>{"status": "completed"});
```

Also in the others kind of request,

In addition I do not know if this is the right syntax, so you need to double check

Reply:
Sure, I'll double check it